tests/keras/layers/convolutional_test.py

```diff
@@ -862,6 +862,45 @@ def test_upsampling_2d():
                 assert_allclose(np_output, expected_out)
 
 
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason="cntk does not support it yet")
```

Comment:
Use `'` for string delimitation, for consistency

Reply:
Of course, my bad.